_posts/2019-03-06-release-1.8.0.md

```diff
@@ -0,0 +1,187 @@
+way. We do this by cleaning up and refactoring Flink under the hood to allow
+more efficient feature development in the future. This includes removal of the
+legacy runtime components that were subsumed in the major rework of Flinks
+underlying distributed system
+([FLIP-6](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=65147077))
+as well as refactorings on the Table API that prepare it for the future
+addition of the Blink enhancements
+([FLINK-11439](https://issues.apache.org/jira/browse/FLINK-11439)).
+
+Nevertheless, this release includes some important new features and bug fixes.
+The most interesting of those we'll highlight below. Please consult the
+[complete changelog](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12315522&version=12344274)
+and the [release notes](https://ci.apache.org/projects/flink/flink-docs-release-1.8/release-notes/flink-1.8.html)
+for more details.
+
+
+## New Features and Improvements
+
+* **Schema evolution support for more data types when restoring savepoints**:
+  Back in Flink 1.7.0, we added support for changing the schema of state when
+  using Avro data types, such as `GenericRecord` or code generated
+  `SpecificRecord`s.
+
+  The pool of data types that support state schema evolution have been expanded
+  to include POJOs and Java Enums. For state types that use POJOs, you can now
+  add or remove fields from your POJO while remaining backwards compatibility.
+  For state types that use Java Enums, Enum fields can also be added or
+  removed, as well as changing types of the Enum fields.
+
+  Moreover, this applies also when you're using these data types within another
+  composite type, such as tuples, `Either`, Scala case classes, etc. For
+  example, the `MyPojo` type in `ValueState<Tuple2<Integer, MyPojo>>` or
+  `ListState<Either<Integer, MyPojo>>`, which is a POJO, is allowed to evolve
+  its schema.
+
+  For a full overview of the list of data types that now support schema
+  evolution as well as their evolution specifications and limitations, please
+  refer to the [State Schema
+  Evolution](https://ci.apache.org/projects/flink/flink-docs-master/dev/stream/state/schema_evolution.html)
+  documentation page.
+
+* **Java-serialization free savepoints**: Previously, when writing the metadata
+  for state in savepoints, the state serializers were also Java-serialized as
+  part of the metadata. This caused several problems, especially for state
+  schema evolution.
+
+  Back in Flink 1.7.0, we introduced new serialization compatibility
+  abstractions for the `TypeSerializer` interface to remove this behaviour. In
+  Flink 1.8.0, all of Flink's built-in `TypeSerializer`s have been upgraded to
+  use the new serialization compatibility abstractions, and therefore will no
+  longer be Java-serialized into savepoints.
+
+  This greatly improves Flink's savepoint interoperability and schema
```

Comment:
"interoperability" with what? I would remove or explain the interoperability. Also the whole phrase can become "This allows the schema of the state stored in Flink's savepoints to evolve".